### DISCUSSION TITLE
Remote-SSH in NvChad

### DISCUSSION THREAD

**Discussion by seacowx** [NvChad/NvChad#2502] [Ideas]
Dear maintainers of NvChad. Thank you for making beautiful neovim more accessible to everyone. I am wondering if you would consider add functionality such as ["Remote-SSH"](https://code.visualstudio.com/docs/remote/ssh) from vscode to NvChad? I work on the server a lot and would really appreciate this functionality to be added to NvChad. Again, thank you for all the amazing work you have done and looking forward to your reply.

**Comment by siduck** [comment]
@SeacowX nvchad is a neovim config so any neovim plugin would work! look for remote ssh plugins for neovim, check awesome-neovim repo

**Reply by seacowx** [reply]
Will do! Thank you for the timely reply!
